README.rst

```diff
@@ -165,31 +165,31 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
-    @software{pygmt_2022_6702566,
+    @software{pygmt_2022_7481934,
       author       = {Uieda, Leonardo and
                       Tian, Dongdong and
                       Leong, Wei Ji and
                       Jones, Max and
                       Schlitzer, William and
                       Grund, Michael and
                       Toney, Liam and
+                      Fröhlich, Yvonne and
                       Yao, Jiayuan and
                       Magen, Yohai and
                       Materna, Kathryn and
-                      Fröhlich, Yvonne and
                       Belem, Andre and
                       Newton, Tyler and
                       Anant, Abhishek and
                       Ziebarth, Malte and
                       Quinn, Jamie and
                       Wessel, Paul},
       title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
-      month        = jul,
+      month        = dec,
       year         = 2022,
       publisher    = {Zenodo},
-      version      = {v0.7.0},
-      doi          = {10.5281/zenodo.6702566},
-      url          = {https://doi.org/10.5281/zenodo.6702566}
+      version      = {v0.8.0},
```

Comment:
Remove the prefix `v` as mentioned in https://github.com/GenericMappingTools/pygmt/pull/2272#issuecomment-1364930311?
```suggestion
      version      = {0.8.0},
```

Ping @GenericMappingTools/pygmt-maintainers for comments on the change.